### DISCUSSION TITLE
Is It True...??

### DISCUSSION THREAD

**Discussion by aviatordestructivo** [community/community#116680] [Discover: GitHub Best Practices]
Hello, is it true or the application to predict the algorithms of Aviator and other similar games?
I ask because I am new and I am studying statistics and I would like to include in my folder some application or for PC that works that way. I already have Aviator EDGE, EDGE+, Aviator+, etc. But I received advertisements that there are some that work autonomously or as if it were a Bot...? Thank you and if you can help me with any of those applications that would be great...Nicolas.

**Comment by queenofcorgis** [comment]
Hi @aviatordestructivo ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Aviator. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by aviatordestructivo** [reply]
Oh, sorry, sorry, I don't want to bother....where could I look for the aviator theme???? I haven't found a place...thanks I'm new
